### DISCUSSION TITLE
JKBMS BLE: IO->Force XXX Off don't seem to work

### DISCUSSION THREAD

**Discussion by mlauss2** [Louisvdw/dbus-serialbattery#983] [General]
Hello,

I have 3 JK B2A20S15P connected bia BLE; they work very well but the "Force XXX Off" buttons don't seem to work.  Is this a known issue?

Thanks!
    Manuel

**Comment by mr-manuel** [comment]
It works only with Daly BMS and LLT/JBD BMS. It's a bug that it is also displayed for the other BMS. The buttons are hidden for all BMS where this is not working in the next version.

**Reply by mlauss2** [reply]
Oh I see.  I thought this corresponds to the 3 switches that the JKBMS app provides in the "Control" section.  Can the jkbms ble code write values back to the BMS?
EDIT: ah no.  And you'd probably need the password as well.
